tests/test_benchmark_filters.py

```diff
@@ -34,18 +37,23 @@
     ],
 )
 def test_filter_datapoints_benchmark(
-    platform: ixmp4.Platform, profiled, benchmark, filters
-):
+    platform: ixmp4.Platform,
+    # TODO: improve type hints for fixtures
+    profiled: Any,
+    benchmark: Any,
```

Comment:
I haven't figured out how to properly annotate these fixtures. Do you know? If not, should we figure this out or accept them as they are now?

Reply:
profiled: probably need to make our own type alias in conftest.py since this is a custom fixture
benchmark: maybe `pytest_benchmark.fixture.BenchmarkFixture` works?

Reply:
Benchmark won't be possible, unfortunately, until https://github.com/ionelmc/pytest-benchmark/issues/212 is fixed. I adapted the `TODO` and added a type alias for profiled as suggested :)